toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -502,6 +520,18 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 		}
 	}
 
+	// Add groups
+	err = addGroups(installChroot, config.Groups)
```

Comment:
What it intentional to move `addGroups` out of the `if !config.IsRootFS()` check?

Reply:
Unclear why we would want to gate groups behind rootfs while we don't with users.